azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/validator/V2ConfigurationValidator.java

```diff
@@ -79,7 +90,7 @@ public String validateWebContainer() {
         if (runtime == null) {
             return "Pleas config the <runtime> in pom.xml.";
         }
-        if (runtime.getWebContainer() == null) {
+        if (WebContainer.fromString(runtime.getWebContainerRaw()) == null && !WebContainerUtils.isJavaSeWebContainer(runtime.getWebContainerRaw())) {
```

Comment:
runtime -> runtimeConfig
getWebContainerRaw -> getWebContainer

Reply:
Validate if runtime.getWebContainer is validate or not should be extracted in One method.

Reply:
As maven plugin also support display name like `Java SE`, differs from `WebContainer` value, so validate both the cases